app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import classname from 'classnames';
+import PropTypes from 'prop-types';
+import {
+  classIconRule,
+  iconRule,
+  isStoryDontEstimated,
+  IsShow,
+  estimateRule,
+  labelSplit,
+  isRelease,
+} from '../../rules/story';
+
+const StoryPoints = () => (
+  <div>
+    <span className="Story__estimate">1</span>
```

Comment:
Two things:

- We could use an array of points here and map it instead of hardcode all the options
- In the near future, we'll receive this array as a prop anyway because the list of possible points is project-dependent (the user can configure it)

Reply:
How would this be? 🤔

Reply:
When you create an project you need to defined how are pointer of the stories.